tikv/raftstore/fsm_apply.go

```diff
@@ -136,7 +186,69 @@ type notifier struct {
 	router *router
 }
 
+type applyContext struct {
+	tag              string
+	timer            *util.SlowTimer
+	host             *CoprocessorHost
+	router           applyRouter
+	notifier         notifier
+	engines          *Engines
+	cbs              util.MustConsumerVec
+	applyTaskResList []ApplyTaskRes
+	execCtx          *applyExecContext
+	wb               *WriteBatch
+	wbLastBytes      uint64
+	wbLastKeys       uint64
+	lastAppliedIndex uint64
+	committedCount   uint64
+
+	enableSyncLog  bool
+	syncLogHint    bool
+	useDeleteRange bool
+}
+
+type waitSourceMergeState struct {
+	pendingEntries []*eraftpb.Entry
+	pendingMsgs    []Msg
+	readyToMerge   *atomic.Uint64
```

Comment:
This library is convenient, we can use it to replace other atomic access values.
But it's a new library, we need to add an entry in `go.mod` file.

Reply:
sorry, I would add this thanks.